tests/fail/tree-borrows/reserved/cell-protected-write.stderr

```diff
@@ -7,11 +7,11 @@ Warning: this tree is indicative only. Some tags may have been hidden.
 | Re*|      │   └────<TAG=callee:x> Strongly protected
 | Re*|      └────<TAG=y,callee:y,caller:y>
 ──────────────────────────────────────────────────────────────────────
-error: Undefined Behavior: write access through <TAG> is forbidden because it is a foreign tag for <TAG>, which would hence change from Permission(Reserved { ty_is_freeze: false }) to Permission(Disabled), but <TAG> is protected
+error: Undefined Behavior: write access through named pointer 'y,callee:y,caller:y' is forbidden because it is a foreign tag for named pointer 'callee:x', which would hence change from Reserved to Disabled, but named pointer 'callee:x' is protected
   --> $DIR/cell-protected-write.rs:LL:CC
    |
 LL |             *y = 1;
-   |             ^^^^^^ write access through <TAG> is forbidden because it is a foreign tag for <TAG>, which would hence change from Permission(Reserved { ty_is_freeze: false }) to Permission(Disabled), but <TAG> is protected
+   |             ^^^^^^ write access through named pointer 'y,callee:y,caller:y' is forbidden because it is a foreign tag for named pointer 'callee:x', which would hence change from Reserved to Disabled, but named pointer 'callee:x' is protected
```

Comment:
Oh, so now the names are also used for errors? Makes sense, but then the docs of the miri extern fn should explain that. Also it might be good to print both the tag and the name? The current wording sounds as if "tagged pointer" and "named pointer" are two fundamental types of pointers. So e.g. "write access through tag <TAG>" and then optionally `(named 'y,callee:y,caller:y')`.